config/sources/families/media.conf

```diff
@@ -59,8 +59,7 @@ case $BRANCH in
 	;;
 
 	edge)
-#		KERNELBRANCH="branch:linux-5.18.y"
-#		KERNELPATCHDIR='media-'$BRANCH
+		KERNELPATCHDIR='media-'$BRANCH
 		KERNELSOURCE='https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git'
```

Comment:
This line is IMO unnecesarry as kernel.org is default source.

Reply:
In the future, I can remove it.